conntrack_linux.go

```diff
@@ -354,38 +363,89 @@ func (f *ConntrackFilter) AddIP(tp ConntrackFilterType, ip net.IP) error {
 	return nil
 }
 
+// AddPort adds a Port to the conntrack filter if the Layer 4 protocol allows it
+func (f *ConntrackFilter) AddPort(tp ConntrackFilterType, port uint16) error {
+	switch f.protoFilter {
+	// TCP, UDP, DCCP, SCTP, UDPLite
+	case 6, 17, 33, 132, 136:
+	default:
+		return errors.New("Filter attribute not available without a Layer 4 protocol")
```

Comment:
Please have error msg also print the `protoFilter` value.

Reply:
Also change 
`[..] without a Layer 4 protocol` => `[...] without a valid Layer 4 protocol` 
either "valid" or "supported", you choose

Reply:
done